gammapy/irf/core.py

```diff
@@ -35,13 +35,14 @@ class IRF:
 
     default_interp_kwargs = dict(bounds_error=False, fill_value=None,)
 
-    def __init__(self, axes, data=0, unit="", meta=None):
+    def __init__(self, axes, data=0, unit="", is_pointlike = False, meta=None):
         axes = MapAxes(axes)
         axes.assert_names(self.required_axes)
         self._axes = axes
         self.data = data
         self.unit = unit
         self.meta = meta or {}
+        self._is_pointlike = is_pointlike
```

Comment:
Maybe treat the `is_pointlike` info as part of the `meta` data? I guess the motivation for the separate argument is to stay independent of the serialisation format (which is good!), however we can introduce arbitrary internal meta data and corresponding keywords as we like. The advantage is that the meta data can be easier copied over to the reduced data as well and we keep the option for a unified data structure of `IRF` and `Map` in future.

Reply:
We could possibly add a table in the docs with "allowed" meta keywords for IRFs and a corresponding description. Like:

| Keyword           | Description                 | Type  |
|-------------------|-----------------------------|-------|
| is_pointlike      | Whether IRF is pointlike    | bool  |
| safe_energy_range | Recommend safe energy range | tuple |